include/swift/RemoteInspection/TypeLowering.h

```diff
@@ -159,6 +159,39 @@ class TypeInfo {
   }
 
   virtual ~TypeInfo() { }
+
+  static bool areEqual(const TypeInfo *LHS, const TypeInfo *RHS) {
```

Comment:
I think this would benefit from a comment why this can't be an operator==()?

Reply:
Since I'm passing in pointers I implemented it as a static function so I could avoid checking for null for both arguments on the call site. I'll add a comment explaining that.